### PR TITLE
🖥️ add call to the emoji list, along with redux

### PR DESCRIPTION
**Pull request by clanam**
This should give us some more space to work by adding the store middleware. We have people, we can be fancy! :D

### CODE REVIEW

**Comment by gmkohler** [comment]
@clanam 📸❓

File: emoji-ui/src/__tests__/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "globals": {
+    "it": "readonly"
```
Comment (nifengle) [inline emoji-ui/src/__tests__/.eslintrc]: what does this lint config do?

Comment (clanam) [reply emoji-ui/src/__tests__/.eslintrc]: it lets the tests use "it" as a global variable. there's probably some other way of doing it but i didn't want to spend time configuring lint.

Comment (clanam) [reply emoji-ui/src/__tests__/.eslintrc]: at least, i didn't want to spend any more time configuring lint than i needed to to get it to find basic syntax errors for me.

**Pull request merged**